### PR TITLE
[Test] Update layer golden test format

### PR DESCRIPTION
**Pull request by zhoonit**
## Commits to be reviewed in this PR

<details><summary>[Test] Update layer golden test format</summary><br />

**Changes proposed in this PR:**
- Add initial_weight to the layer golden data
- Add layer::build() point to the translayer
- Properly call layer instead of  __call__ in translayer
- Run formatter(black)

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>

</details>

### CODE REVIEW

File: test/input_gen/recorder.py [hunk 1]
```diff
@@ -175,8 +175,8 @@ def train_step(model, optimizer, loss_fn, initial_input, label, writer_fn, **kwa
 
         # loss = loss_fn(label, outp[-1])
         loss = []
-        if kwargs.get('multi_out', None) != None:
-            multi_out = kwargs.get('multi_out', [])
+        if kwargs.get("multi_out", None) != None:
```
Comment (kparichay) [inline test/input_gen/recorder.py:178]: this is minor but single quotes is more pythonic :)

**Review by kparichay — APPROVED** [review]
LGTM

File: test/input_gen/recorder.py (hunk 1 above)
Comment (zhoonit) [reply test/input_gen/recorder.py:178]: I am using black [formatter](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#strings). We'd better have a unified formatter :)

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Pull request merged**